tests/admin_views/test_autocomplete_view.py

```diff
@@ -94,6 +94,32 @@ def test_custom_to_field(self):
             'pagination': {'more': False},
         })
 
+    def test_custom_to_field_permission_denied(self):
+        Question.objects.create(question='Is this a question?')
+        request = self.factory.get(self.url, {'term': 'is', **self.opts, 'field_name': 'question_with_to_field'})
+        request.user = User.objects.create(
+            username='staff', password='secret', email='super@example.com', is_staff=True
+        )
```

Comment:
We can reuse `self.user`:
```suggestion
        request.user = self.user
```

Reply:
How so? I need a staff user since I am checking for a specific permission.

Reply:
@codingjoe `setUpClassData` gives us a staff user: 

```python
        cls.user = User.objects.create_user(
            username='user', password='secret',
            email='user@example.com', is_staff=True,
        )
```